test/kubectx.bats

```diff
@@ -25,7 +35,14 @@ load common
   [[ "$output" = "error: No previous context found." ]]
 }
 
-@test "create one context and list contexts" {
+@test "list contexts when no kubeconfig exists" {
```

Comment:
this is currently not immune to my compute having a ~/.kube/config

please deliberately do `KUBECONFIG=./non-existing-file kubectx` here. it should work the same way.

Reply:
there is a export KUBECONFIG=... in setup(), in common.bash, I understand that it is equivalent

Reply:
Ah right, but doesn’t that get changed or something if we reorder the tests?

It would be good to explicitly set a kubeconfig to a nonexisting file to test this without side effects of the outside world.